langchain/src/tools/google_calendar/descriptions.ts

```diff
@@ -0,0 +1,24 @@
+export const CREATE_TOOL_DESCRIPTION = `A tool for creating Google Calendar events and meetings.
+
+INPUT example:
+"action": "google_calendar_create",
```

Comment:
Output parser is the standard way of doing this:

https://js.langchain.com/docs/modules/prompts/output_parsers/#structured-output-parser

Reply:
Ah ok, my mistake!